src/documentation/reference/errors/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
considering using `index.mdx` for the first page of a section instead of `overview.mdx` because:
- it would allow us to make this page's URL `stellar.org/developers/docs/api/errors` 
- it would indicate what styling should be applied to this page's H1 title

Reply:
Was just contemplating this myself. We'll have to special-case `index`, but I think that's a good idea